src/index.ts

```diff
@@ -29,6 +30,8 @@ async function route(request: Request) {
 			const args = text.split(" ");
 
 			if (args.length === 1 && !args[0]) {
+				return help();
```

Comment:
help() should also be returned on `/shorten help`

Reply:
STALE!